### ISSUE TITLE
[select] a11y not announcing in Safari with VoiceOver

### ISSUE THREAD

**Issue by DRiFTy17** [tyler-technologies-oss/forge#136]
**Describe the bug:**
The accessibility features of the select component are not being announced at with VoiceOver in Safari.

**To Reproduce:**
1. Enable VoiceOver on Mac
2. View the select demo in Storybook here.
3. Attempt to navigate the component via keyboard and not that the options are not announced in the dropdown.

**Expected behavior:**
The options should be announced in the dropdown when navigating view keyboard, as well as when an option is selected.

**Please complete the following information:**
 - Forge version: ^2.0.0
 - I have searched existing issues before creating this report? Y
 - Browser: Safari
 - Platform: Desktop
 - OS: Mac

**Additional context:**
Very similar issue noted [here](https://developer.apple.com/forums/thread/706788) on the Apple developer forum.

Works fine in Chrome in Firefox with VoiceOver.

**Comment by DRiFTy17** [comment]
Fixed in 3.0
